c2cgeoportal/scaffolds/create/+package+/templates/ngeo.html_tmpl

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="\{\{mainCtrl.lang\}\}" ng-app="app" ng-controller="MainController as mainCtrl">
```

Comment:
Why are the backslashes necessary? I am asking the question because we don't have backslashes in geoportailv3. https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportailv3/templates/index.html